typings/index.d.ts

```diff
@@ -4316,12 +4318,9 @@ export type ThreadChannelResolvable = ThreadChannel | Snowflake;
 
 export type ThreadChannelTypes = 'GUILD_NEWS_THREAD' | 'GUILD_PUBLIC_THREAD' | 'GUILD_PRIVATE_THREAD';
 
-export interface ThreadCreateOptions<AllowedThreadType> {
-  name: string;
-  autoArchiveDuration: ThreadAutoArchiveDuration;
+export interface ThreadCreateOptions<AllowedThreadType> extends StartThreadOptions {
```

Comment:
Shouldn't this be:
```suggestion
export interface ThreadCreateOptions<AllowedThreadType extends ThreadChannelTypes> extends StartThreadOptions {
```
?

Reply:
No becauseThreadChannelTypes contains types that AllowedThreadTypes doesn’t